sceptre/strides/config/prod/github-oidc-sage-bionetworks-synapse-login.yaml

```diff
@@ -0,0 +1,17 @@
+template:
+  type: http
+  url: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.7.6/templates/IAM/github-oidc-provider.j2
+stack_name: github-oidc-sage-bionetworks
+Dependencies:
+  - prod/github-oidc-provider.yaml
+parameters:
+  ProviderRoleName: github-oidc-sage-bionetworks
```

Comment:
this should match stack_name but needs to be less than 64 chars

also you'll need to create a follow on PR to update GH action role-to-assume for sceptre-stridespool, https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/.github/workflows/main.yaml#L159C1-L166